src/content.js

```diff
@@ -1,4 +1,31 @@
 // this is the code that "Runs" the extension.
 // This is seperate from the code that runs the UI inside the extension.
-
 const deliberatelybroke = true;
+
+const videoTags = $('meta[name=\'keywords\']').attr('content').toLowerCase();
```

Comment:
I think I would prefer if these were closer in code to the check that used them, but that's a personal thing and I'm not sure it's even worse mentioning.